lib/elixir/lib/task.ex

```diff
@@ -341,10 +339,12 @@ defmodule Task do
   Receives a group of tasks and a message and finds
   a task that matches the given message.
 
-  This function returns a tuple with the returned value
-  in case the message matches a task that exited with
-  success alongside the matching task. It returns `nil`
-  if no task was found. It exits if the task has failed.
+  If the message matches a task that finished normally,
+  this function returns a tuple with the task's
+  result and `Task` struct. If the message matches a
+  task that terminated abnormally, this function will exit.
```

Comment:
It need not be abnormally.
```elixir
Task.async(fn() -> exit(:normal) end)
```
Will cause the task to exit normally and cause an exit. It should be fine just to remove "abnormally".